src/components/ConfirmCard/index.tsx

```diff
@@ -2,8 +2,10 @@
 
 import { Backdrop, Box, Button, Card, Typography } from '@mui/material';
 import { Dispatch, SetStateAction } from 'react';
+import { DictProps } from '@i18n/i18n';
+import { useDictionary } from '@/lib/i18n/hooks';
```

Comment:
We should stick to using either the ts path alias @i18n (defined in tsconfig, more compact) or just plain @